CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributors
+
+Hello Contributors!  Thx for your interest! We're so glad you're here. 
+
+## Important Resources
+  - bugs: [https://github.com/tinkerbell/tinkerbell.org/issues](https://github.com/tinkerbell/tinkerbell.org/issues)
+  - feature requests: [https://github.com/tinkerbell/tinkerbell.org/issues](https://github.com/tinkerbell/tinkerbell.org/issues)
+
+### Code of Conduct
+Available via [https://github.com/tinkerbell/tinkerbell.org/blob/master/.github/CODE_OF_CONDUCT.md](https://github.com/tinkerbell/tinkerbell.org/blob/master/.github/CODE_OF_CONDUCT.md)
```

Comment:
Can you please update this to https://github.com/tinkerbell/.github/blob/master/CODE_OF_CONDUCT.md